kernel/sql-federation/optimizer/src/main/java/org/apache/shardingsphere/sqlfederation/optimizer/converter/segment/expression/impl/BinaryOperationExpressionConverter.java

```diff
@@ -72,6 +72,11 @@ private static void register() {
         register(SqlStdOperatorTable.CONCAT);
         register(SqlStdOperatorTable.PATTERN_ALTER);
         register(SqlStdOperatorTable.MOD);
+        register(OperatorConverter.DIV);
```

Comment:
Can you rename this calss to SQLExtensionOperatorTable?

Reply:
Done